manifests/init.pp

```diff
@@ -22,6 +22,12 @@
 # $zonefilepath::               Directory containing zone files
 #
 # $localzonepath::              File holding local zones like RFC1912 or RFC1918 files.
+#                               The special value 'unmanaged' can be used if one plans
+#                               to create custom RFC1912/RFC1918 zones via ::dns,
+#                               where the inclusion of package-shipped zone files is
+#                               not desired.
+#
+# $defaultzonepath::            File holding default zone includes like db.local.
```

Comment:
This is in params.pp but not exposed in init.pp. It should either be exposed or removed from the documentation. Currently the isn't managed either so it doesn't exist. That causes the acceptance test to fail (named won't start up on EL7).